services/web/server/src/simcore_service_webserver/config/server-docker-prod.yaml

```diff
@@ -68,6 +68,10 @@ storage:
   host: ${STORAGE_HOST}
   port: ${STORAGE_PORT}
   version: v0
+catalog:
+  host: catalog
+  port: 8000
+  version: v0
```

Comment:
no variables?

Reply:
no. Actually is more informative than actual configurations. E.g. all fastapi based servers are port 8000 and all aiohttp 8080.